nav2_recoveries/include/nav2_recoveries/recovery.hpp

```diff
@@ -173,12 +173,14 @@ class Recovery : public nav2_core::Recovery
       // TODO(orduno) #868 Enable preempting a Recovery on-the-fly without stopping
       if (action_server_->is_preempt_requested()) {
         RCLCPP_ERROR(
-          node_->get_logger(), "Received a preemption request for %s,"
-          " however feature is currently not implemented. Aborting and stopping.",
+          node_->get_logger(), "Received a preemption request for %s,",
```

Comment:
This can be an info now (or even a debug after you get it working)

Reply:
Sorry, I missed that one. Do you think is better an info or debug?

Reply:
Lets do debug